client/src/components/Notifications/Notification.tsx

```diff
@@ -1,4 +1,45 @@
+import { theme } from "consts/theme";
 import React, { useEffect, useState } from "react";
+import styled, { StyledComponent } from "styled-components";
+
+const Notifications = styled.div`
```

Comment:
I think this should be singular:

```suggestion
const Notification = styled.div`
```